pandas/core/arrays/timedeltas.py

```diff
@@ -227,6 +244,18 @@ def _validate_fill_value(self, fill_value):
                              "Got '{got}'.".format(got=fill_value))
         return fill_value
 
+    @property
+    def is_monotonic_increasing(self):
+        return algos.is_monotonic(self.asi8, timelike=True)[0]
+
+    @property
+    def is_monotonic_decreasing(self):
+        return algos.is_monotonic(self.asi8, timelike=True)[1]
+
+    @property
+    def is_unique(self):
+        return len(unique1d(self.asi8)) == len(self)
```

Comment:
is this *only* for TimedeltaArray? the index should not be doing this

Reply:
I much prefer comments in the code to non-obvious things.

Reply:
I'll add comments to that effect